docs/site/Mixin.md

```diff
@@ -74,6 +75,7 @@ And define mixin `LoggerMixin`:
 {% include code-caption.html content="src/mixins/logger.mixin.ts" %}
 
 ```ts
+import {MixinTarget} from '@loopback/core';
```

Comment:
We probably should mention a workaround for the TypeScript limitation when a method needs to be overridden. See https://github.com/strongloop/loopback-next/blob/688cbee82fb18c347252081ea7651b0fb3e43053/packages/core/src/mixin-target.ts#L25. See https://loopback.io/doc/en/lb4/apidocs.core.mixintarget.html.

Reply:
Also mention:
```ts
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
```